doc/man5/config.pod

```diff
@@ -438,6 +438,14 @@ generators will use.  Other random bit generators ignore this name.
 This sets the property query used when fetching the random bit generator and
 any underlying algorithms.
 
+=item B<seed>
+
+This sets the randomness source that should be used.
```

Comment:
Perhaps mention the default is "SEED-SRC" and link to that man page?

Reply:
Added a note that is the default for all except the FIPS provider (which uses the callbacks unconditionally).